addons/common/monitoring/prometheus/src/main/java/org/kie/kogito/monitoring/prometheus/common/rest/MetricsResource.java

```diff
@@ -15,11 +15,9 @@
  */
 package org.kie.kogito.monitoring.prometheus.common.rest;
 
-import org.kie.kogito.monitoring.prometheus.common.PrometheusRegistryProvider;
-
-public abstract class MetricsResource {
+/**
+ * Needed only to notify AddonsConfigDiscovery about prometheus availability
```

Comment:
We used [a similar comment](https://github.com/kiegroup/kogito-runtimes/blob/main/addons/common/persistence/postgresql/src/main/java/org/kie/kogito/persistence/KogitoProcessInstancesFactory.java#L25) in other addons 
```suggestion
 * This class must always have exact FQCN as <code>org.kie.kogito.monitoring.prometheus.common.rest. MetricsResource </code>
```